src/descriptor.rs

```diff
@@ -148,10 +152,145 @@ impl Descriptor {
     }
 
     /// Constructs a new [`Descriptor`] from a byte [`Vec`].
+    ///
+    /// # Warning
+    ///
+    /// Users are advised to use the `new_*` methods whenever possible.
     pub fn from_vec(bytes: Vec<u8>) -> Result<Self, DescriptorError> {
         Self::from_bytes(&bytes)
     }
 
+    /// Constructs a new [`Descriptor`] from an `OP_RETURN` payload.
+    ///
+    /// The payload is expected to be less than 80 bytes.
```

Comment:
The payload _must_ be _no more than_ 80 bytes (it can hit the limit).

Reply:
Done in 4044d26126bab31c892e72782827c655e12511dd